Server/Controllers/MatchController.cs

```diff
@@ -258,7 +258,28 @@ public async Task<ActionResult> After([FromBody] AfterMatchRequest request)
 
             return Ok();
         }
-
+        [HttpPost]
+        [Route("patreon-player-update")]
+        public async Task<ActionResult> PatreonPlayerUpdate([FromBody] PatreonPlayerUpdate request)
+        {
+            foreach (var player in request.players)
+            {
+                var ctx = _context.Players.FirstOrDefault(p => p.SteamId.ToString() == player.SteamId);
```

Comment:
`ctx` isn't really what it is, plus that's an abbreviation that makes code harder to read. It's better 
 to rename it to `player` and name the loop variable `playerUpdate`